src/main/java/Duke.java

```diff
@@ -1,10 +1,20 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String name = "Stella";
+        System.out.println("Hello! I'm " + name);
+        System.out.println("What can I do for you?");
+        System.out.println("-----------------------------------");
```

Comment:
can make this line a void function so that the code looks less messy as this line appears many times